tests/e2e/default/analytics/analytics.wdio-spec.js

```diff
@@ -12,11 +12,6 @@ const chtConfUtils = require('@utils/cht-conf');
 const chtDbUtils = require('@utils/cht-db');
 const { TARGET_MET_COLOR, TARGET_UNMET_COLOR } = analyticsPage;
 
-const updateSettings = async (settings) => {
-  await utils.updateSettings(settings, 'api');
-  await commonPage.sync(true);
-  await browser.refresh();
-};
 
 const compileTargets = async (targetsFileName = 'targets-config.js') => {
```

Comment:
Please move this method inside the first `describe`.

And on L39, please remove the `roles: ['chw']` because the `role` is already set as `cht`, in the [userFactory](https://github.com/medic/cht-core/blob/master/tests/factories/cht/users/users.js#L24)